ethcore/vm/src/schedule.rs

```diff
@@ -291,6 +294,7 @@ impl Schedule {
 	/// Schedule for the Istanbul fork of the Ethereum main net.
 	pub fn new_istanbul() -> Schedule {
 		let mut schedule = Self::new_constantinople();
+        schedule.have_chain_id = true;
```

Comment:
```suggestion
		schedule.have_chain_id = true;
```

Reply:
This is the dumb OSX merge conflict review tool I'm using. It ignores whitespace settings. :/